--- src/Makefile.sav	2001-10-08 11:57:06.000000000 +0200
+++ src/Makefile	2005-11-16 14:15:01.000000000 +0100
@@ -3,14 +3,16 @@
 JFLEX     = jflex -skel skeleton.nested
 CUP       = java java_cup.Main 
 
-JAVAC     = javac -target 1.1 -bootclasspath /usr/lib/jdk1.1/lib/classes.zip
+JAVAC     = javac -target 1.4 -bootclasspath /usr/lib/java/jre/lib/rt.jar
 #JAVAC     = jikes   
 JAR       = jar
 
+JAVADOC   = javadoc
+
 DEST      = ../lib
 
 # jdk 1.2 
-CLPATH    = -classpath /usr/lib/java/jre/lib/rt.jar:.
+CLPATH    = -classpath /usr/lib/java/jre/lib/rt.jar:.
 
 # jdk 1.1
 #CLPATH    = -classpath /usr/lib/java/lib/classes.zip:.
@@ -61,3 +63,7 @@
 	cd $(DEST); $(JAR) -c0mf manifest $(JARFILE) JFlex java_cup
 	rm -rf $(DEST)/java_cup $(DEST)/JFlex $(DEST)/manifest
 
+javadoc: 
+	$(JAVADOC) -sourcepath . -d ../api JFlex
+
+
